packages/features/form-builder/FormBuilder.tsx

```diff
@@ -120,6 +120,13 @@ export const FormBuilder = function FormBuilder({
               : field.getOptionsAt
               ? dataStore.options[field.getOptionsAt as keyof typeof dataStore]
               : [];
+
+            // Note: We recently started calling getAndUpdateNormalizedValues in the FormBuilder. It was supposed to be called only on booking pages earlier.
+            // Due to this we have to meet some strict requirements like of labelAsSafeHtml.
+            if (fieldsThatSupportLabelAsSafeHtml.includes(field.type)) {
```

Comment:
@Amit91848 Changed the logic to handle all fields that support safeHtml and not just **boolean** type